### PR TITLE
Update helm repo url

### PR DESCRIPTION
**Pull request by AntonOfTheWoods**
This URL appears to work well, as discussed in https://github.com/coredns/deployment/issues/240.

### CODE REVIEW

**Review by chrisohaver — APPROVED** [review]

**Comment by AntonOfTheWoods** [comment]
@chrisohaver , sorry for the bother but this may have been blocked by me not having a signoff in my commit message. That has been fixed now but that may have stopped auto merge. Or maybe PRs require two signoffs?

**Comment by haad** [comment]
I think much better would be to actually enable github pages in this repository and use published URL. But this way we can doc working.

**Pull request merged**
